Web Dev/Backend/Languages/Go.md

```diff
@@ -0,0 +1,98 @@
+> ## GOLang
+
+## Why learn GoLang
+
+- Go (or Golang) is a programming language designed for building fast, reliable, and efficient software at scale. Its simplicity, concurrency features, and built-in garbage collection make it particularly well-suited for building large-scale, networked applications and distributed systems. Additionally, Go's growing popularity and strong community support make it an increasingly relevant language for both web and backend development.
+
+- Go is extensively used by google,uber,netflix,twitch,dropbox etc, go is mainly used in building api's , go is known for its extensbility in microservice architecture, lately go's popularity has been increasing so it is a very good choice to learn go.
+
+- Go (or Golang) can be used for a variety of applications, including building web servers and APIs, creating networked and distributed systems, developing command-line tools and utilities, writing system software and device drivers, and more generally for any application that requires high performance, reliability, and scalability.
+
+---
+
+## 🎓 Take Away Skills
+
+After completing this learning path you will be knowledgeable in:
+
+- Operators and Data Types
+- Loops and Conditional Statements
+- Arrays , Slice etc
+- Functions in GO lang
+- Struct, Map in golang
+- Interface in golang
+- Methods in go lang
+- Defer Panic and Recover
+- Pointers in golang
+
+---
+
+## 🧑🏻‍💻 Programming Knowledge
+
+There is no prerequesite in learning go lang , but familarity of basic programming is good
+
+## 📲 Installation and Setup
+
+- Download Go: Visit the official Go website at golang.org and download the appropriate installer for your operating system.
```

Comment:
- Add hyperlink for Go Website in **line 35**
- Add the preferred Code editors or IDE for Go development in **line 43**